### PR TITLE
Deeplink samples feature merging csharp

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/tab-deeplink/csharp/DeepLinkBot/Manifest/manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿{
-  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
-  "manifestVersion": "1.12",
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
```
Comment (ChetanSharma-msft) [inline samples/tab-deeplink/csharp/DeepLinkBot/Manifest/manifest.json:2]: Please remove duplicate sample

File: samples/tab-deeplink/csharp/DeepLinkBot/Manifest/manifest.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  "manifestVersion": "1.16",
   "version": "1.0.0",
```
Comment (ChetanSharma-msft) [inline samples/tab-deeplink/csharp/DeepLinkBot/Manifest/manifest.json:4]: Please run build pipeline as well.

File: samples/tab-deeplink/csharp/DeepLinkBot/Manifest/manifest.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
   "id": "<<YOUR-MICROSOFT-APP-ID>>",
   "packageName": "com.microsoft.deeplinkbot",
```
Comment (ChetanSharma-msft) [inline samples/tab-deeplink/csharp/DeepLinkBot/Manifest/manifest.json:6]: I can't see Readme updates for combined features.

File: samples/tab-deeplink/csharp/DeepLinkBot/Views/Home/ExtendedDeepLinks.cshtml [hunk 4]
```diff
@@ -226,10 +157,87 @@ microsoftTeams.app.openLink("https://teams.microsoft.com/l/app/1542629c-01b3-4a6
...
+                    <div class="card-body">
+                        <div class="card-header">
+                            <b>Navigate to chat (Personal chat with bot)</b>
+                        </div>
+                        <div class="configure-header">
+                            To use this option, configure the function using following
+                            parameters
+                            <!-- Since we are using pre tag for having the code snippet in specified format ignoring the indentation here -->
+                            <pre>
+microsoftTeams.app.openLink(`https://teams.microsoft.com/l/entity/f6661eaa-0cf9-4eb2-93b1-58278abaf4d4/conversations`);
```
Comment (ChetanSharma-msft) [inline samples/tab-deeplink/csharp/DeepLinkBot/Views/Home/ExtendedDeepLinks.cshtml:203]: What is this? f6661eaa-0cf9-4eb2-93b1-58278abaf4d4

Hardcoded value?

Please check in nodejs sample as well.

File: samples/tab-deeplink/csharp/README.md [hunk 5]
```diff
@@ -69,6 +69,9 @@ urlFragment: officedev-microsoft-teams-samples-tab-deeplink-csharp
     git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
     ```
    - Update channelID placeholer `<AddYourTeamsChannelId>` value in your `DeepLinkBot.cs` file with your particular channel id from any Team in Teams. (You can get it manually by clicking on 3 dots in any team's channel and fetch it's link and extract the channel id ) like `General` channel) Example: `19:cbe3683f25094106b826c9cada3afbe0@thread.skype`
+
+   - Navigate to `wwwroot/js/env.js` file and update your AppId at placeholder `<<App-ID>>` (You can get it manually frrm [teams admin portal](https://admin.teams.microsoft.com/).
+	- ![personal-AddedBot ](DeepLinkBot/images/TeamsAdminPortal.png)
    
    - Update the `appsettings.json` configuration for the bot to use the MicrosoftAppId, MicrosoftAppPassword, BaseURL ,
```
Comment (ChetanSharma-msft) [inline samples/tab-deeplink/csharp/README.md:76]: Update Readme file for descriptions and capabilities.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments.

File: samples/tab-deeplink/csharp/DeepLinkBot/Views/Home/ExtendedDeepLinks.cshtml (hunk 4 above)
Comment (Prithvi-MSFT) [reply samples/tab-deeplink/csharp/DeepLinkBot/Views/Home/ExtendedDeepLinks.cshtml]: Yes removed it

File: samples/tab-deeplink/csharp/README.md (hunk 5 above)
Comment (Prithvi-MSFT) [reply samples/tab-deeplink/csharp/README.md:76]: updated readme

File: samples/tab-deeplink/csharp/DeepLinkBot/Manifest/manifest.json (hunk 3 above)
Comment (Prithvi-MSFT) [reply samples/tab-deeplink/csharp/DeepLinkBot/Manifest/manifest.json:6]: updated readme

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Comment by ChetanSharma-msft** [comment]
Please execute build pipeline as well.

File: samples/tab-deeplink/csharp/assets/sample.json [hunk 6]
```diff
@@ -3,10 +3,10 @@
     "name": "officedev-microsoft-teams-samples-tab-deeplink-csharp",
     "source": "officeDev",
     "title": "DeepLink",
-    "shortDescription": "Microsoft Teams sample app for demonstrating deeplink from Bot chat to Tab consuming Subentity ID",
+    "shortDescription": "Microsoft teams sample which displays deeplinks for call, video, chat, navigate to app and navigate within tab pages which are supported for tab and bot.",
```
Comment (Wajeed-msft) [inline samples/tab-deeplink/csharp/assets/sample.json:6]: It should be Microsoft Teams and not teams.

File: samples/tab-deeplink/csharp/README.md [hunk 7]
```diff
@@ -69,6 +69,9 @@ urlFragment: officedev-microsoft-teams-samples-tab-deeplink-csharp
     git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
```
Comment (Wajeed-msft) [inline samples/tab-deeplink/csharp/README.md:69]: Do we need to update all the screenshots as well?

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

File: samples/tab-deeplink/csharp/assets/sample.json (hunk 6 above)
Comment (Prithvi-MSFT) [reply samples/tab-deeplink/csharp/assets/sample.json]: Yes changes it

File: samples/tab-deeplink/csharp/README.md (hunk 7 above)
Comment (Prithvi-MSFT) [reply samples/tab-deeplink/csharp/README.md:69]: Updated images and gif's

File: samples/tab-deeplink/csharp/DeepLinkBot/Manifest/manifest.json (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/tab-deeplink/csharp/DeepLinkBot/Manifest/manifest.json:2]: PR link for removing duplicate sample
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/723

File: samples/tab-deeplink/csharp/DeepLinkBot/Manifest/manifest.json (hunk 2 above)
Comment (Prithvi-MSFT) [reply samples/tab-deeplink/csharp/DeepLinkBot/Manifest/manifest.json:4]: Yes all builds passed

**Comment by Prithvi-MSFT** [comment]
Build status  - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/4264621655

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**